blocks/blog-posts/blog-posts.js

```diff
@@ -0,0 +1,120 @@
+import {
+  createOptimizedPicture, loadCSS,
+  readBlockConfig,
+  toClassName,
+} from '../../scripts/lib-franklin.js';
+
+const defaultAuthorName = 'Cognizant Netcentric';
+const defaultAuthorTitle = '';
+const defaultAuthorImage = '/icons/nc.svg';
+
+export function buildCard(card) {
+  const {
+    path, title, image, tags, profiles: authorProfile,
+  } = card;
+
+  const cardElement = document.createElement('article');
+  cardElement.classList.add('teaser');
+
+  if (authorProfile.Image === '') authorProfile.Image = defaultAuthorImage;
+
+  cardElement.innerHTML = `
+    <p class="tags">${JSON.parse(tags).join(', ')}</p>
+    <a href="${path}" target="_self" class="teaser-link">
+      <h2 class="teaser-description">
+        ${title}
+      </h2>
+    </a>
+    <div class="authorprofile-container">
+      <div class="authorprofile-image">
+        <div class="nc-image-base">
+            <div class="nc-image-container " itemscope="" itemtype="http://schema.org/ImageObject">
+                <img class="nc-image" src="${authorProfile.Image ?? defaultAuthorImage}" itemprop="contentUrl" alt="" sizes="10vw" />
```

Comment:
Isn't `authorProfile.Image` already checked in line 19?

Reply:
I checked twice because `Image` can be `undefined` or `''` depending on what is stored in the index. I think this may be a quirk with the importer that has yet to be ironed out @mhaack . For now I could consolidate the two checks into one.